pandas/tests/test_downstream.py

```diff
@@ -47,6 +47,17 @@ def test_xarray(df):
     assert df.to_xarray() is not None
 
 
+def test_xarray_cftimeindex_nearest():
```

Comment:
need a skip if ctime is not installed

Reply:
need a skip for xarray not installed (do these as decorators rather than inside the function)

Reply:
Can you point me to an example for how you prefer to go about this in pandas?  I was following the skipping style used in the rest of `test_downstream.py` (e.g. using the `import_module` function defined within it).  I'm happy to convert to using decorators if that's desired.

Reply:
grep for something like this

```td.skip_if_no("numba", "0.46.0"))````

Reply:
Perfect, thanks!